nifi-nar-bundles/nifi-avro-bundle/nifi-avro-processors/src/test/java/org/apache/nifi/processors/avro/TestConvertAvroToJSON.java

```diff
@@ -266,6 +297,34 @@ public void testMultipleAvroMessages() throws IOException {
     }
 
     @Test
+    public void testMultipleAvroMessagesStandardJson() throws IOException {
```

Comment:
The test method name says StandardJson but the USE_AVRO_JSON property is set to true in the test